Pages/login.html

```diff
@@ -0,0 +1,140 @@
+  </head>
+  <body>
+    <!-- Navbar starts  -->
+    <nav id="navbar">
+      <div class="navbar-left">
+        <a href="../index.html"><h3 class="h3"><span class="primary-color">Men</span>Stack</h3></a>
+      </div>
+
+      <div class="navbar-right">
+        <a href="../Pages/signup.html"
+          ><button class="btn-link btn btn-primary">Register</button></a
+        >
+      </div>
+    </nav>
+
+    <!-- Hero starts -->
+    <section id="hero">
+      <span class="overlay"></span>
+      <form class="input-wrapper">
+        <div class="form-title">
+          <h5 class="h5">LOGIN</h5>
+        </div>
+        <label class="label" for="email">Email/Username:</label>
+        <input
+          class="text-input border-default"
+          type="email"
+          name="email"
+          id="email"
+          placeholder="rahul@menStack.com"
+          required
+        />
+
+        <label class="label" for="username">Password:</label>
+        <input
+          class="text-input border-default"
+          type="password"
+          name="password"
+          id="password"
+          placeholder="**********"
+          required
+        />
+
+        <a href="#"><span for="tnc" class="span-sm">Forgot Password?</span></a>
+
+        <div class="cta">
+          <button class="btn btn-primary">Login</button>
+        </div>
+
+        <div class="login-icons-wrapper">
+          <p class="p-sm">Login using:</p>
+          <ion-icon class="login-icon" name="logo-google"></ion-icon>
+          <ion-icon class="login-icon" name="logo-apple"></ion-icon>
+          <ion-icon class="login-icon" name="logo-facebook"></ion-icon>
+        </div>
+      </form>
+    </section>
+
+    <footer id="footer">
+      <div class="footer">
+        <div class="footer-1">
+          <h3 class="h3 primary-color">
+            Men<span class="secondary-color-light">Stack</span>
+          </h3>
+
+          <div class="contact-wrapper">
+            <ion-icon class="footer-icon" name="call"></ion-icon>
+            <p class="p-sm">+91 9876543210</p>
+          </div>
+          <div class="contact-wrapper">
+            <ion-icon class="footer-icon" name="mail"></ion-icon>
+            <p class="p-sm">care@menstack.com</p>
+          </div>
+          <div class="icons-wrapper">
+            <ion-icon class="footer-icon" name="logo-twitter"></ion-icon>
+            <ion-icon class="footer-icon" name="logo-facebook"></ion-icon>
+            <ion-icon class="footer-icon" name="logo-linkedin"></ion-icon>
+            <ion-icon class="footer-icon" name="logo-github"></ion-icon>
+          </div>
+        </div>
+        <div class="footer-2">
+          <p class="p">Get to Know Us</p>
+          <p class="p-xs">About Us</p>
+          <p class="p-xs">Careers</p>
+          <p class="p-xs">Press Releases</p>
+          <p class="p-xs">Gift a Smile</p>
+          <p class="p-xs">Menstack Science</p>
+          <p class="p-xs">Menstack Cares</p>
```

Comment:
just a minor change, use list tag for listing things.

Reply:
Thank you Gaurav for pointing this out. I have used this as a footer component in all pages so I will have to make changes everywhere as of now 🙊 . I will definitely change this in next update for the app.